.pre-commit-config.yaml

```diff
@@ -50,9 +50,9 @@ repos:
       log_file: '.artifacts/flake8.pycodestyle.log'
     - id: lint-requirements
       name: lint-requirements
-      entry: bin/lint-requirements
+      entry: python -m tools.lint_requirements sentry
```

Comment:
This is because we're deleting lint-requirements and have created a better version that takes into consideration the new requirements files.